src/app/components/AccountPage/AccountSettings.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Heading, Link, LinkTypes } from '@nypl/design-system-react-components';
+
+import appConfig from '../../data/appConfig';
+
+const AccountSettings = ({ patron }) => {
+  const { legacyCatalog } = appConfig;
+
+  return (
+    <div className="account-settings">
+      <div>
+        <Heading level={3} text="Personal Information" />
+        <Link
+          href={`${legacyCatalog}/patroninfo*eng~Sdefault/${patron.id}/modpinfo`}
+          type={LinkTypes.Default}
+          className="edit-link settings"
+          attributes={{
+            target: '_blank',
+          }}
+        >
+          Edit
+        </Link>
+      </div>
+      <hr />
+      <dl>
+        <dt>Telephone</dt>
+        <dd>{patron.phones ? patron.phones[0].number : null}</dd>
```

Comment:
My phone number is empty, which seems to cause the <dd> to collapse
<img width="931" alt="Screen Shot 2021-01-15 at 10 07 32 PM" src="https://user-images.githubusercontent.com/439581/104795693-58aa8c80-577e-11eb-991a-92a6e959eb3e.png">

Reply:
Good find. I have a fix for this. But I am checking with Ellen first about whether in this case the field should be displayed if there is no value.

Reply:
Ellen decided we will say 'None' for missing info. Just added that change

Reply:
Works for me